docs/src/DeveloperDocumentation/styleguide.md

```diff
@@ -190,6 +190,54 @@ end
 However, as always, rules sometimes should be broken.
 
 
+## Optional arguments for parents of return values
+
+Several objects in OSCAR have `parent`s, e.g. polynomials, group elements, ... 
+Whenever a function creates such objects from an input which does not involve 
+the output's parent, the user should have the possibility to pass on this 
+parent as an optional argument. 
```

Comment:
So here it specifically talks about an "optional" argument. But then in the example it shows *two* methods for specifying the caller, only one involving an optional argument. Nothing seems to resolve this?